api/topo/topo.proto

```diff
@@ -31,10 +31,8 @@ service Topo {
   rpc Read (ReadRequest) returns (ReadResponse) {
   }
 
-  // Represents the bidirectional stream between
-  // onos-topo and a client for the purpose of
-  // - streaming notifications
-  rpc StreamChannel (stream StreamMessageRequest) returns (stream StreamMessageResponse) {
+  // Subscribe returns a stream of topo change notifications
+  rpc Subscribe (SubscribeRequest) returns (stream SubscribeResponse) {
```

Comment:
I am not sure if it should be bidirectional or not but at least in gNMI/p4runtime it is bidirectional.

Reply:
@adibrastegarnia - you are right - gNMI/p4 stream channel is bi-directional. I didnt have any use case at present for it to be bidirectional. We can make it bi-directional when we have a use case.